.env.production.sample

```diff
@@ -251,6 +251,9 @@ SMTP_FROM_ADDRESS=notifications@example.com
 # Maximum allowed character count
 MAX_TOOT_CHARS=500
 
+# Maximum allowed hashtags to follow in a feed column
```

Comment:
I would add a note that it can have significant performance implications

Reply:
That's a reasonable suggestion, but I'm unclear on exactly how the performance would be impacted. Does going from 4 to 8 double the potential load? quadruple? How does adding a hashtag compare to adding a followed user?

Reply:
The issue is not so much with the “posts having any of these tags” mode but “posts having all these tags”

Reply:
Note added.